src/main/java/uk/gov/hmcts/ecm/common/model/helper/Constants.java

```diff
@@ -44,7 +44,7 @@ private Constants() {
     public static final String UPDATING_STATE = "Updating";
     public static final String TRANSFERRED_STATE = "Transferred";
     public static final String TRANSFERRED_IN_STATE = "TransferredIn";
-    public static final String IN_PROGRESS = "In progress";
+    public static final String IN_PROGRESS = "inProgress";
```

Comment:
Is there a reason for it being renamed in camel case? It doesnt seem to follow the style of the other constants here

Reply:
Yes, in et-sya-front we have direct mapping from values  to hublink colors. So in case I want to leave the 'In progress' value I have to create mapping in frontend(we have there enums, constants with  'inProgress' etc).